File: C programming/Chapter 8(Strings)/Q4.c
// Write a program to print the number of vowels and consonants in a string
#include <stdio.h>

int main()
{
    char str[100];
    int vowel = 0, consonats = 0, i;
    printf("Enter a String:-");
    gets(str);
    for (i = 0; str[i] != '\0'; i++)
    {
        if (str[i] == 'A' || str[i] == 'a' || str[i] == 'E' || str[i] == 'e' || str[i] == 'O' || str[i] == 'o' || str[i] == 'I' || str[i] == 'i' || str[i] == 'U' || str[i] == 'u')
            vowel++;
        else
            consonats++;
    }
    printf("The number of vowels is %d and number of consonants is %d", vowel, consonats);
    return 0;
}

File: Python Tutorials/Assignment 6/Q4.py
# Write a program to count the number of occurrences of any item from the tuple with a function count_occurrences. (Take tuple input from the user)

def count_occurences(t1, item):
    count = 0
    for i in t1:
        if item == i:
            count = count+1
    return f"The occurence of {item} is {count}"


t1 = tuple(input("Enter the elements in a tuple seperated by comma:").split(","))
item_in_tuple = input("Enter the item in a tuple to find the occurence:")
print(count_occurences(t1, item_in_tuple))
